### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BoukeHaarsma23**
The order of the colors are different for my Loki zero.

Updated the readme also to increase readability.

### CODE REVIEW

**Comment by pastaq** [comment]
If course they are -_-.

I'm need to reflect that in the driver.

File: README.md [hunk 1]
```diff
@@ -117,15 +117,15 @@ To change modes:
...
-brightness: The current average luminosity of all three colors in the range [0-255]`.
+- brightness: The current average luminosity of all three colors in the range `[0-255]`.
 Setting this to `0` will turn off the LED's while setting it to `255` represents the
 maximum luminosity of the selected multi_intensity range.
-multi_index: Displays the order of the colors when setting multi_intensity. Set to
-`red green blue`.
-multi_intensity: The current intensity of each individual color [0-255] where 0 is off
-and 255 is most luminous. The value is dispalyed and set with three numbers separated
+- multi_index: Displays the order of the colors when setting multi_intensity. Set to
+`red blue green`.
```
Comment (pastaq) [inline README.md:126]: This isn't university true. The Max has this as described here. I will reach out to AYN to see if they can make them all the same order.

Comment (dmanlfc) [reply README.md:126]: yeah it's annoying as you have to reverse the values. thanks @pastaq

**Comment by pastaq** [comment]
Resolved in #9. I had the registers assigned wrong.

**Pull request closed without merging**